ext/cgimap/cgimap.cpp

```diff
@@ -339,6 +420,20 @@ boost::shared_ptr<output_buffer> hijack_rack_request::get_buffer_internal() {
 void hijack_rack_request::finish_internal() {
 }
 
+
+boost::posix_time::ptime hijack_rack_request::get_current_time() const {
+
+  return boost::posix_time::second_clock::local_time();
+}
+
+/*
+
+const std::string hijack_rack_request::get_payload() {
```

Comment:
That's a left over from testing with feature/bulk_upload branch, which could be deleted for now.